src/views/layouts/master.twig

```diff
@@ -27,7 +27,8 @@
 | store.settings.rating_enabled                      | bool     | Is one of: store_enabled or shipping_enabled or products_enabled                                         |
 | store.settings.arabic_numbers_enabled              | bool     | Does the merchant wants the numbers to be shown as Arabic format                                         |
 | store.settings.is_multilingual                     | bool     |                                                                                                          |
-| store.settings.currencies_enabled                  | bool     |                                                                                                          |
+| store.settings.currencies_enabled                  | bool     |     
+| store.settings.google_map_key                      | string   |                                                                           |
```

Comment:
no need for this line, we are passing the key directly via the twilight event

Reply:
@dhafer95 @eslamoo